static/js/components/PersonalTab.js

```diff
@@ -30,7 +35,7 @@ class PersonalTab extends React.Component {
       <Grid className="profile-tab-grid">
         <Cell col={1} />
         <Cell col={10}>
-          <PersonalForm {...this.props} />
+          <PersonalForm {...this.props} validator={personalValidation} />
```

Comment:
we're pulling `validator` out of `this.props` in the `onChange methods in the various input field methods - so we could import `personalValidation` in `PersonalForm` and set `defaultProps`. I originally was using this pattern for the components where we want the same validation in all contexts, but I had a lot of trouble getting this to type-check (I think because `PersonalForm` and the other form components subclass `ProfileFormFields`), so I went with this pattern instead.